saleor/discount/forms.py

```diff
@@ -42,15 +42,18 @@ def clean(self):
         if 'voucher' in cleaned_data:
             voucher = cleaned_data['voucher']
             try:
-                discount = get_voucher_discount_for_checkout(
+                discount_amount = get_voucher_discount_for_checkout(
```

Comment:
Again, "amount" seems to indicate a number while this (I assume) returns a Money object.